config/opensearch_dashboards.yml

```diff
@@ -184,3 +184,6 @@
 # Set the value of this setting to false to suppress search usage telemetry
 # for reducing the load of OpenSearch cluster.
 # data.search.usageTelemetry.enabled: false
+
+# Set the value of this setting to true to fetch data from multiple OpenSearch clusters
+# data_source.enabled: false
```

Comment:
Is this config currently being used anywhere? If not, it's better to add it along with other related changes in another PR, so when people "blame" the code on github, they will not land on some PR/issue that has not enough context

Reply:
let's not modify the yml config file, we have set defualt to false in the plugin config

Reply:
removed